### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by AlexIoannides**
Following feedback from the initial launch:

- a diagram illustrating how Bodywork operates was created and injected into the `Overview`, `Key Concepts` and `User Guide` sections;
- the `Overview` was modified so that it no longer assumes that the reader is using Kubernetes, but that they might be orchestrating containers on services such as Azure/Amazon Container Service; and,
- the `Quickstart` was re-worked to make it easier to follow if the example repo hasn't been investigated first. Consequently it has much better 'flow' and can be read stand-alone and still make sense.

### CODE REVIEW

**Comment by AlexIoannides** [comment]
[Overview.pdf](https://github.com/bodywork-ml/bodywork-core/files/5649785/Overview.pdf)
[Quickstart.pdf](https://github.com/bodywork-ml/bodywork-core/files/5649786/Quickstart.pdf)

**Comment by AlexIoannides** [comment]
visual_explanation

File: README.md [hunk 1]
```diff
@@ -2,18 +2,28 @@
...
+The process of containerising machine learning code using Docker to build images, pushing the build artefacts to an image repository and then configuring a container orchestration platform to run batch workloads and deploy services, requires skills and expertise that most machine learning engineers do not have the time (and often the desire) to learn.
 
-- automate the configuration of Kubernetes jobs and deployments to run complex machine learning workflows that result in machine learning service deployments.
-- continuously deliver machine learning code - for training models and defining model-scoring services - directly from your Git repository into running containers on Kubernetes.
+Scale this scenario to one where there are multiple models to worry about, all needing to be re-trained and re-deployed, and it is easy to see how quickly the management of these pipelines will become a large and undesirable burden.
+
+This is where the Bodywork MLOps framework steps-in - to take care of delivering your code to the right place and executing it at the right time, so that your models are always trained, deployed and available to the rest of your product team.
```
Comment (Marios85) [inline README.md:17]: "to the rest of your product team". I think it sounds better to end on available, as well there's no assumption on whom the model is being made available to.

File: README.md [hunk 2]
```diff
@@ -2,18 +2,28 @@
...
+
+![bodywork_diagram](https://bodywork-media.s3.eu-west-2.amazonaws.com/visual_explanation.png)
+
+Bodywork is a tool built upon the Kubernetes container orchestration platform and is aimed at machine learning engineers to help them:
+
+- **continuously deliver code** - for training models and defining model-scoring services, by having running containers on Kubernetes pull it directly from your project's Git repository, and so removing the need to build-and-push container images; and,
```
Comment (Marios85) [inline README.md:23]: "by having running" - Use either having or running, not both.  Or maybe "using"?

"and so" - Therefore or thereby would sound better. 

Remove trailing "and,"

File: README.md [hunk 3]
```diff
@@ -2,18 +2,28 @@
...
+- **automate deployments** - of batch workloads and model-scoring services, using the Bodywork workflow-controller to orchestrate end-to-end machine learning workflows on Kubernetes.
+
+Kuberetes is available as a managed service from all major cloud computing platforms.
```
Comment (Marios85) [inline README.md:26]: Typo in k8s

**Review by Marios85 — CHANGES_REQUESTED** [review]
I've made some inline suggestions in Readme.md which would also need to be applied to Index.md. Otherwise it all looks good and reads very well.

File: README.md (hunk 1 above)
Comment (AlexIoannides) [reply README.md:17]: Agreed.

File: README.md (hunk 2 above)
Comment (AlexIoannides) [reply README.md:23]: - **Continuously deliver code** - for training models and defining model-scoring services. Bodywork containers running on Kubernetes will pull code directly from your project's Git repository, removing the need to build-and-push your own container images.

Comment (AlexIoannides) [reply README.md:23]: This is much better - thanks.

File: README.md (hunk 3 above)
Comment (AlexIoannides) [reply README.md:26]: Nice one.

**Comment by AlexIoannides** [comment]
All sorted - as always, greatly improved 🙂

**Review by Marios85 — APPROVED** [review]
Cheers, looks good! :)

**Review by Marios85 — APPROVED** [review]

**Pull request merged**